students/Pirouz_N/lesson06/test_mailroom.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python3
+"""
+Purpose: Mailroom Part 4 testing mail room functionality python certificate from UW
+Author: Pirouz Naghavi
+Date: 07/14/2020
+"""
+
+import pytest
+import os
+
+
+@pytest.fixture
+def mailroom_gen_db_seven_donors():
+    import mailroom
+    mailroom.donor_db = {
+        'William Gates, III': [653772.32, 12.17],
+        'Jeff Bezos': [877.33],
+        'Paul Allen': [663.234, 43.87, 1.32],
+        'Mark Zuckerberg': [1663.23, 4300.87, 10432.0],
+        'Jayz': [999.9, 100.0, 100.0],
+        'Beyonce': [1000.0, 100.0],
+        'Melinda Gates': [10000000.0, 1000000.0, 1000020.0],
+    }
+    mailroom.continue_loop = True
```

Comment:
I don't believe this variable is in your module anymore

Reply:
Oh true! Thank you I missed that.